eng/EnsureGlobalJsonSdk.ps1

```diff
@@ -167,5 +181,28 @@ if (Test-Path $globalJson) {
             Write-Verbose "global.json updated"
             Write-Verbose (Get-Content $globalJson -Raw)
         }
+
+        if ($SdkVersionOverride -and $AdditionalNuGetFeeds) {
+            $nugetConfig = Join-Path (Get-Item $globalJson).Directory.FullName 'NuGet.config'
+            if (Test-Path $nugetConfig) {
+                Write-Verbose "Additional Feeds requested - Updating NuGet.config"
+                Write-Verbose "Feeds are..."
+                $AdditionalNuGetFeeds | %{
+                    $feed = $_
+                    Write-Verbose "`t$feed"
+                }
+
+                # Download NuGet.exe 
+                $NuGetExe = join-path $env:TEMP 'nuget.exe'
+                if (-not (Test-Path $NuGetExe -PathType Leaf)) {
+                    Invoke-WebRequest https://dist.nuget.org/win-x86-commandline/latest/nuget.exe -OutFile $NuGetExe
+                    Write-Verbose "Downloaded nuget.exe to $NuGetExe"
+                }
+
+                for ($i = 0; $i -lt $AdditionalNuGetFeeds.Count; $i++) {
+                    & $NuGetExe Sources Add -Name "transient-delete-$i" -Source $AdditionalNuGetFeeds[$i]
```

Comment:
Under MSBuild of preview versions this can't find vswhere.  Does this override the nuget.config instead of adding to it?

Reply:
Hmm, this might just be a local build error for me.

Reply:
Nevermind this is a problem with installing VsWhere, not with the nuget feeds.  Not sure why this is happening on my system.